plop/hooks/index.js

```diff
@@ -0,0 +1,24 @@
+module.exports = plop => {
+  plop.setGenerator("Hook Story", {
+    description: "New hook story",
+    prompts: [
+      {
+        type: "input",
+        name: "componentName",
+        message: "What is the name of your hook? (use spaces if multi word)"
+      }
+    ],
+    actions: [
+      {
+        type: "add",
+        path: "src/hooks/__stories__/{{camelCase componentName}}.stories.mdx",
+        templateFile: "plop/hooks/hook-story.txt"
+      },
+      {
+        type: "add",
+        path: "src/hooksDummyComponents/{{properCase componentName}}.jsx",
```

Comment:
I kind of not understand this flow. Why we create a dummy component for a new hook?  Is this only fot the meta and props component in the story?

Reply:
the ArgsTable can accept different settings, but unless it receives a component.
So in order for an easier way to populate the hook's arguments, I created a dummy component which we will edit and config the arguments